### PR TITLE
Filter out inactive communities

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Comment by robbiecarlton** [comment]
@levity doesn't seem to be breaking anything. Can I merge?

File: api/controllers/CommunityController.js [hunk 1]
```diff
@@ -118,7 +119,7 @@ module.exports = {
 
   joinWithCode: function (req, res) {
     var community
-    return Community.query('whereRaw', 'lower(beta_access_code) = lower(?)', req.param('code')).fetch()
+    return Community.query('whereRaw', 'lower(beta_access_code) = lower(?) and active=true', req.param('code')).fetch()
```
Comment (levity) [inline api/controllers/CommunityController.js]: for the sake of cleanliness, can you switch this to the function syntax and have the active=true bit in a normal where?

File: api/models/Post.js [hunk 2]
```diff
@@ -238,7 +239,7 @@ module.exports = bookshelf.Model.extend({
       'communities.users.communities',
       'user'
     ]})
-   .then(post => {
+    .then(post => {
```
Comment (levity) [inline api/models/Post.js]: indentation

File: api/policies/checkAndSetMembership.js [hunk 3]
```diff
@@ -1,45 +1,46 @@
...
-      } else {
-        sails.log.debug(format("policy: checkAndSetMembership: fail. user %s, community %s",
-          req.session.userId, community.id));
-        res.forbidden();
-      }
-    });
-  });
-
-};
+      .then(function (membership) {
```
Comment (levity) [inline api/policies/checkAndSetMembership.js]: indentation

**Comment by levity** [comment]
looks good otherwise

**Pull request merged**